spec/dummy/app/avo/resources/course_resource.rb

```diff
@@ -8,6 +8,7 @@ class CourseResource < Avo::BaseResource
 
   field :id, as: :id
   field :name, as: :text
+  field :skills, as: :tags, blacklist: -> { record.skill_blacklist }, suggestions: -> { record.skill_suggestions }
```

Comment:
Suggestion: Change this API for this field from `blacklist` to `disallowed`

Reply:
Otherwise, how's is the field working? Does it fix your need?